.circleci/config.yml

```diff
@@ -160,6 +161,13 @@ commands:
                   - run:
                       name: "Waiting for << parameters.wait >>"
                       command: riot -v run 'wait' << parameters.wait >>
+            # - unless:
```

Comment:
Should this be removed?  If not, we should have a comment on why its commented out